src/Uno.UI/UI/Xaml/Controls/Popup/PopupPanel.cs

```diff
@@ -174,17 +174,18 @@ protected override Size ArrangeOverride(Size finalSize)
 			return finalSize;
 		}
 
-#if __ANDROID__
```

Comment:
Why was this added to be android only, originally ?

Reply:
I actually added it a couple of days ago to fix ComboBox theme updates, I limited it to Android out of conservatism because I only actually needed it on Android (the Parent was being set to the native host view but not being unset).